### ISSUE TITLE
部署面：gauss 冒烟 + sugon 8 卡验收 + 文档口径

### ISSUE THREAD

**Issue by ACautomata** [ACautomata/cynosure#201]
## Parent

#197（ADR-0016 · 预训练 torchrun 化）

## What to build

真机验收与部署文档：sugon-deploy skill 的 pretrain 段更新——torchrun 8 卡启动命令（`CYNOSURE_PG_TIMEOUT_MIN` + 显式 --run-dir），替换现行「pretrain 不走 torchrun、检测到 RANK 显式拒绝」的说法；gauss 2×A6000 冒烟（CUDA 栈上的多 rank 路径）；sugon 8×DCU 真机验收——完整 pretrain run、四态控制流无挂死、记录 coronal 级条件单步墙钟（预期 ≈ ÷8，参照单卡 71 分钟）；文档口径标注：单卡 run 与多卡 run 的预训练读数不可横向比较逐位值（8 卡重放锚为「同 config + 同 world_size 逐位一致」，卷积算法尾差 ~1e-12 量级统计等价）。

## Acceptance criteria

- [ ] sugon-deploy skill pretrain 段更新：8 卡 torchrun 命令 + --run-dir + PG timeout
- [ ] gauss 2×A6000 冒烟通过（CUDA 多 rank 路径）
- [ ] sugon 8 卡完整 pretrain run：无挂死、gate 语义正确、产物齐全
- [ ] coronal 级单步墙钟实测记录（对照单卡 71 分钟，验证 ≈÷8）
- [ ] 单/多卡读数不可横向比较的口径标注落文档

## Blocked by

- #199
- #200

**Comment by ACautomata** [comment]
## 8 卡 pretrain 真机验收（2026-09-22 完成）

run：`20260922-issue201-8card/pretrain_run`（sugon 8×DCU）。启动命令与本 PR 更新的 sugon-deploy 步骤 5 一致：

```bash
CYNOSURE_PG_TIMEOUT_MIN=40 torchrun --nproc_per_node=8 -m cynosure.cli pretrain \
  --config <run>/config.json --run-dir <run>/pretrain_run
```

### 验收对照

| AC | 结果 |
|---|---|
| gauss 2×A6000 冒烟（CUDA 多 rank 路径） | ✅ `tests/test_pretrain_distributed.py` 4/4 全绿（37.04s）：四态控制流、DDP 逐位一致、告警归并、usage 契约 |
| sugon 8 卡完整 pretrain run | ✅ 100/100 步走满，全程无挂死、无 watchdog 触发，log 尾部 8 rank 干净退出 |
| 产物齐全 | ✅ `pretrain_report.json` + `checkpoints/pretrain_discriminator.pt`（10.6 MB，sha256 入 provenance）+ `metrics.jsonl` 100 条 pretrain 事件 |
| gate 语义正确 | ✅ 门槛 0.65：11 条件 recon-AUC 0.4978–0.5068（全部 chance 带）→ `gate_passed=false`、白名单空、终止原因「步数上限耗尽」——判别器 100 步密集训练未出 chance 带（与 t174 单卡同形态），train 侧按硬前置拒绝开跑，语义正确 |
| 文档口径（单/多卡读数不可横向比较） | ✅ 落 sugon-deploy skill 步骤 6 + `docs/spec/reward-model.md`（PR #206） |

报告含 11 条件 per-condition AUC 与条件卷数（512 卷 ×10 + mra 10 卷）、六件套数据口径指纹（real pool / held-out / channel stats / 判别器 config / ckpt / 条件词汇表，各带 sha256）。`broadcast_flag` 四态分发全程工作，rank0 单点判定 + 全 rank 步调一致。

### coronal 单步墙钟（AC4）与全条件对照

单卡基线 = t174 run 逐条件 `elapsed_s`；8 卡 = 本 run 逐条件均值（n≈9 步/条件，11 条件轮转）。

| 条件 | 单卡 (min/步) | 8 卡 avg (min/步) | 加速比 |
|---|---|---|---|
| **t1w/coronal** | **70.9–71.2** | **10.49**（3 次读数 10.7/10.7/10.4） | **÷6.8** |
| flair/coronal | 34.8 | 5.17 | ÷6.7 |
| t2w/axial | 45.6 | 5.22 | ÷8.7 |
| t1w/axial | 13.2 | 1.28 | ÷10.3 |
| swi/axial | 13.1 | 1.27 | ÷10.3 |
| t1w/sagittal | 7.6 | 1.33 | ÷5.7 |
| mra/all-planes | 1.1 | 0.84 | ÷1.3（轻条件，通信/调度占比高） |
| flair/sagittal | 35.5 | 15.91（9 步全部 15.8–16.2） | **÷2.2 ⚠️** |
| t2w/sagittal / t2w/coronal / flair/axial | （无单卡基线） | 1.32 / 1.36 / 1.27 | — |

- **coronal ÷6.8，未达理想 ÷8 的结构性因素**：gather 后 rank0 串行重算全局 recon-AUC（非分片段）+ 每 rank 小 batch ODE 重构效率 + 共享实例间歇负载抖动（`CYNOSURE_PG_TIMEOUT_MIN=40` 所针对的同款现象）。
- **⚠️ flair/sagittal 系统性偏离**：9 步全部 ~16 min（÷2.2），与同家族 flair/coronal（÷6.7）不一致，占总墙钟 410 min 中的 143 min。不影响本次验收判据，但归因值得另开票（假说：该条件卷尺寸/重构成本在分片后倾斜，需对照单卡同条件剖析）。
- **总墙钟 6.84 h**（100 步 370 min + 收尾 ~40 min）。收尾段 = 步数耗尽路径的 11 条件最终补测 + 报告/checkpoint 写出，不产 metrics 事件——监控上表现为最后一步后 ~40 min 静默后正常完成退出，符合「补测不进事件流」口径。
- 报告 `condition_auc` 为补测值，与步内最后事件快照同量级（如 t1w/axial 0.5052 = step 99 快照）。

### 遗留提请

1. `docs/spec/orchestration.md` 的 train 编排口径仍是「单实例 4 卡」，实例已升 8 卡——train 侧本次按 4 卡口径未动，分隢单独处理。
2. flair/sagittal 8 卡加速比归因（建议 follow-up ticket）。
